setup.py

```diff
@@ -38,12 +38,19 @@ def get_output(*args, **kwargs):
 
 # get the compile and link args
 kc = "krb5-config"
+autodetect_kc = True
 posix = os.name != 'nt'
 
 # Per https://docs.python.org/3/library/platform.html#platform.architecture
 # this is the preferred way of determining "64-bitness".
 is64bit = sys.maxsize > 2**32
 
+kc_env = 'KRB5CONFIG'
```

Comment:
This is suspiciously close to `KRB5_CONFIG` which is an env var that MIT (maybe Heimdal as well) uses to specify the `krb5.conf` file to use. Would it be better to use the `GSSAPI_` prefix we use for the other env var configs like `GSSAPI_KRB5CONFIG` to avoid any confusion?

Reply:
That's good feedback. I chose `KRB5CONFIG` because [that's what the FreeBSD ports machinery uses for the purpose](https://cgit.freebsd.org/ports/tree/Mk/Uses/gssapi.mk?id=7229781c12bb3ffe8a8f9fa0f6fb2805802951e2#n138). Since my intent with this patch is to contribute something more generally useful, I would be glad to tailor to suit.